react/package.json

```diff
@@ -13,16 +13,15 @@
   },
   "devDependencies": {
     "@apollo/react-testing": "^3.1.3",
-    "@types/jest": "^25.1.2",
-    "@types/node": "^13.7.1",
-    "@types/react": "^16.9.19",
-    "@vtex/test-tools": "^3.0.3",
-    "@vtex/tsconfig": "^0.4.3",
+    "@types/jest": "^25.1.4",
+    "@types/node": "^13.9.8",
+    "@types/react": "^16.9.31",
+    "@vtex/test-tools": "^3.1.0",
+    "@vtex/tsconfig": "^0.4.4",
     "apollo-cache-inmemory": "^1.6.5",
-    "eslint-config-vtex-react": "^6.1.1",
```

Comment:
Will it find the package in the root directory?

Reply:
Yep. We're concentrating the tooling packages in the root